src/lumigo_opentelemetry/__init__.py

```diff
@@ -121,6 +123,7 @@ def init() -> Dict[str, Any]:
 
     tracer_provider = TracerProvider(
         resource=tracer_resource,
+        sampler=LUMIGO_SAMPLER,
```

Comment:
where the magic is supposed to happen

Reply:
> I wouldn't create the sampler as a constant. Just initialize an instance upon usage

@saartochner i feel this way is cleaner, it's also in line with the samplers defined in `opentelemetry.sdk.trace.sampling`